packages/vue/src/router.ts

```diff
@@ -72,76 +70,105 @@ export function vueRouterInstrumentation(
         name: WINDOW.location.pathname,
         op: 'pageload',
         origin: 'auto.pageload.vue',
-        tags,
-        data: {
+        attributes: {
```

Comment:
I'd say it's ok to stop sending this as a tag and start sending it as an attribute - I don't think anybody will depend on this too much...? Does not seem too relevant for a user.

Reply:
agreed. I made the same change in the SvelteKit `browserTracingIntegration`. Realistically, we probably don't need it at all, given that we have `sentry.origin` but it doesn't hurt us to send it as an attribute.

Reply:
IMO we can just remove it now that we have `sentry.origin` across the board, let's save some bundle size.

That was the purpose of having the `routing.instrumentation` tag in the first place

Reply:
I'll remove it 🚀